packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/preview/index.ts

```diff
@@ -75,7 +75,6 @@ export default class CardPayPrepaidCardWorkflowPreviewComponent extends Componen
     }
   }
 
-  // TODO: Refactor long method
```

Comment:
On second thought - the method is mostly just a sequence of tasks so even if we split it up into smaller methods it will still be just that 😀. So I'm fine with this method even when it's a big longer than usual. I did simplify it a bit though.